mappings/net/minecraft/client/particle/ParticleLavaEmber.mapping

```diff
@@ -1,7 +1,7 @@
-CLASS none/bni net/minecraft/client/particle/ParticleLava
+CLASS none/bni net/minecraft/client/particle/ParticleLavaEmber
```

Comment:
Visually it's an ember though, and a number of mods use it as such.

Doesn't really matter, either way.